data.json

```diff
@@ -694,5 +694,11 @@
     "githubUsername": "ChinmayMhatre",
     "imageUrl": "https://user-images.githubusercontent.com/51131670/111893373-5091f600-8a28-11eb-8e94-48b23eec0495.PNG",
     "issueNumber": 403
-  }
+  },
+   {
```

Comment:
This bracket should line up like the rest.
```suggestion
  {
```

Reply:
Could you move the bracket so it lines up with the rest. Just to make the style consistant. :+1:

Reply:
Done